README.md

```diff
@@ -110,14 +111,12 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let publisher = service.publisher().create()?;
 
-    while !SignalHandler::termination_requested() {
+    while let ElkEvent::Tick = Elk::wait(CYCLE_TIME) {
```

Comment:
Out of curiosity. Is `Elk` and `ElkEvent` just a placeholder or the final name?

Reply:
For now I would leave the name like this. When we do the final renaming it may become either `Iox` or maybe `i2x`

Reply:
I'll add an issue for an final API review before the initial release

Reply:
... I added it to #3